public/app/stores/roomStore.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var React = require('react');
+var Reflux = require('reflux');
+
+var roomActions = require('../actions/roomActions');
+
+// TODO: Is this the correct naming style for global variables?
```

Comment:
Well, we do not have the naming convention for global vars as **there must not be any global vars** in app code.

The only case we have where global variable is needed - `config` sharing between backend and frontend